rusk-wallet/src/wallet/gas.rs

```diff
@@ -10,10 +10,16 @@
 use crate::currency::Lux;
 
 /// The minimum gas limit
-pub const MIN_LIMIT: u64 = 350_000_000;
+pub const MIN_LIMIT: u64 = 50_000_000;
 
 /// The default gas limit
-pub const DEFAULT_LIMIT: u64 = 500_000_000;
+pub const DEFAULT_LIMIT: u64 = 100_000_000;
+
+/// The default gas limit for a contract deployment
+pub const DEFAULT_LIMIT_DEPLOYMENT: u64 = 50_000_000;
+
+/// The default gas limit for staking
+pub const DEFAULT_LIMIT_STAKE: u64 = 2_900_000_000;
```

Comment:
Looking at the PR a bit closer, this limit is not only being used for staking operations but also other contract-calls. The variable should hence be called something like `DEFAULT_LIMIT_CONTRACT_CALL` (or shorter `DEFAULT_LIMIT_CALL`)